Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -808,14 +809,35 @@ void ConsoleVariableItemDelegate::setModelData(QWidget* editor, QAbstractItemMod
 {
     if (auto* doubleEditor = qobject_cast<AzQtComponents::SliderDoubleCombo*>(editor))
     {
+        QRegExp rx("((-?\\d+)(\\.\\d+)?)");
+        if (!rx.exactMatch(QString::number(doubleEditor->value())))
+        {
+            QMessageBox::information(NULL, "Title", "Enter data of the <double> type", QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes);
```

Comment:
The message boxes should be parented to `AzToolsFramework::GetActiveWindow()` instead of `NULL`, and should be given a more descriptive title.